.prow.yaml

```diff
@@ -289,6 +289,92 @@ presubmits:
                 path: id_rsa
               - key: id_rsa.pub
                 path: id_rsa.pub
+  - name: e2e-kubeadm-ubuntu-jammy
+    annotations:
+    labels:
+    run_if_changed: '^e2e/'
```

Comment:
Should we refine these. I feel it's overkill running all of these jobs every time there is a change made anywhere in the e2e dir

Reply:
I am up for suggestions, (As a side note, this job is optional for PR merge at the moment)

Reply:
@efiacor @vjayaramrh that's my feeling as well, this however would require modularising each job as they are now defined in single place. I've created issue for that: https://github.com/nephio-project/sig-release/issues/47